config/import-control-test.xml

```diff
@@ -3,7 +3,9 @@
     "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
     "https://checkstyle.org/dtds/import_control_1_4.dtd">
 
-<import-control pkg="com.puppycrawl.tools.checkstyle">
+<import-control
+        pkg="(com\.(puppycrawl\.tools|google|sun)\.checkstyle(\.test)?|org\.checkstyle)"
```

Comment:
Maybe it is better write full packages? like `(com\.puppycrawl\.tools\.checkstyle|com\.sun\.checkstyle\.test...`
it will take more place, but will be easier to understand

Reply:
done